packages/components/src/Autocomplete/tests/Autocomplete.setup.tsx

```diff
@@ -125,9 +127,13 @@ export function Wrapper<T extends OptionLike>({
       value={value}
       onChange={onChange ?? setValue}
       inputValue={inputValue}
-      onInputChange={onInputChange ?? setInputValue}
```

Comment:
would you mind walking me through this change? it doesn't seem equivalent.

aren't we always going to call `setInputValue` now?